eldritch/monsters.py

```diff
@@ -208,8 +214,9 @@ def Hound():
   )
 
 
-def Maniac():  # TODO: custom class when we add globals
-  return Monster("Maniac", "normal", "moon", {"evade": -1, "combat": 1}, {"combat": 1}, 1)
+class Maniac(Monster):  # TODO: custom class when we add globals
```

Comment:
Maybe change the TODO to reflect what's missing - the combat difficulty and damage need to change based on the state.

Reply:
I think this no longer needs a TODO...